vnext/templates/cpp-app/windows/MyApp/pch.h

```diff
@@ -0,0 +1,35 @@
+// pch.h : include file for standard system include files,
+// or project specific include files that are used frequently, but
+// are changed infrequently
+//
+
+#pragma once
+
+#include "targetver.h"
+
+#define NOMINMAX 1
+#define WIN32_LEAN_AND_MEAN 1
+#define WINRT_LEAN_AND_MEAN 1
+
+// Windows Header Files
+#include <windows.h>
+
+#pragma push_macro("GetCurrentTime")
+#undef GetCurrentTime
+
+// Playground pch.h
```

Comment:
Remove reference to Playground

Reply:
There's lots of potential cleanup available that may be subsumed by larger refactors anyway, but tracking bug here: https://github.com/microsoft/react-native-windows/issues/12179